packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -116,7 +123,12 @@ class Metrics {
         buildType: 'prod',
         name: 'endpoint',
         networkType: 'unknown',
-        userAgent: this.getUserAgentInfo()
+        userAgent: this.getUserAgentInfo(),
+        clientInfo: {
+          clientType: options.clientType,
+          os: 'linux', // TODO: BUG AND DOESNT ACCEPT OSNAME ON CA -> bowser.osname || 'null'
```

Comment:
Is there a JIRA to track this TODO work?

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-122143